iterators.tex

```diff
@@ -4188,6 +4285,55 @@
 \end{codeblock}
 \end{itemdescr}
 
+\rSec4[common.iter.op.iter_move]{\tcode{iter_move}}
+
+\indexlibrary{\idxcode{iter_move}!\idxcode{common_iterator}}%
+\indexlibrary{\idxcode{common_iterator}!\idxcode{iter_move}}%
+\begin{itemdecl}
+friend rvalue_reference_t<I> iter_move(const common_iterator& i)
+  noexcept(@\seebelow@)
+    requires InputIterator<I>;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\requires \tcode{!i.is_sentinel}.
+
+\pnum
+\effects Equivalent to: \tcode{return ranges::iter_move(i.iter);}
+
+\pnum
+\remarks The expression in \tcode{noexcept} is equivalent to:
+\begin{codeblock}
+\end{codeblock}
+noexcept(ranges::iter_move(i.iter))
+\end{itemdescr}
+
+\rSec4[common.iter.op.iter_swap]{\tcode{iter_swap}}
+
+\indexlibrary{\idxcode{iter_swap}!\idxcode{common_iterator}}%
+\indexlibrary{\idxcode{common_iterator}!\idxcode{iter_swap}}%
+\begin{itemdecl}
+template <IndirectlySwappable<I> I2>
+  friend void iter_swap(const common_iterator& x, const common_iterator<I2>& y)
+    noexcept(@\seebelow@);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\requires \tcode{!x.is_sentinel \&\& !y.is_sentinel}.
+
+\pnum
+\effects Equivalent to \tcode{ranges::iter_swap(x.iter, y.iter)}.
```

Comment:
The rules are:
* Equivalent to `<expression>`.
* Equivalent to: `<statement>`
* Equivalent to:
  ```c++
  code;
  block;
  ```